person_capture/gui_app.py

```
@@ -8247,6 +8356,27 @@ def _invoke_curator(self) -> None:
+        try:
+            conn_type = (
+                QtCore.Qt.ConnectionType.QueuedConnection
+                if hasattr(QtCore.Qt, "ConnectionType")
+                else QtCore.Qt.QueuedConnection
+            )
+            QtCore.QMetaObject.invokeMethod(
+                worker,
+                "disable_hdr_passthrough",
+                conn_type,
+            )
+        except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)